.codeclimate.yml

```diff
@@ -0,0 +1,21 @@
+engines:
+	rubocop:
+		enabled: true
+	golint:
+		enabled: treu
+	eslint:
+		enabled: true
+	csslint:
+		enabled: true	
+ratings:
+	paths:
+	-app/**
+	-lib/**
```

Comment:
It looks like you copied these directories from an example. We have no `lib` directory, `app` directory, or `.go` files. We also don't want to rate the Ruby files in this repo.